src/utilities/core/Compare.cpp

```diff
@@ -74,19 +75,17 @@ bool IstringPairCompare::operator()(const std::pair<std::string, std::string>& x
   return result;
 }
 
-VersionString::VersionString(const std::string& version)
-  : m_str(version)
-{
+bool VersionString::parseVersionString(const std::string& version) {
```

Comment:
Are these changes to VersionString all cosmetic formatting changes, or is there a functional change hidden in here?